Jellyfin.Api/Controllers/PackageController.cs

```diff
@@ -0,0 +1,119 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+using Jellyfin.Api.Constants;
+using MediaBrowser.Common.Updates;
+using MediaBrowser.Model.Updates;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Jellyfin.Api.Controllers
+{
+    /// <summary>
+    /// Package Controller.
+    /// </summary>
+    [Route("Packages")]
+    [Authorize]
```

Comment:
I believe every route should be limited to admin accounts.

Reply:
I agree that they should be limited, but `PackageService` didn't have all the endpoints requiring admin authentication.

Reply:
Hmm that's interesting. I would need to look through and double check that the web client is the only one using this service, but this is one of those cases where I think it would be fine to tweak the existing API slightly.

Reply:
@dkanada Should I go ahead and update the Controller to limit access to only admin accounts?

Reply:
If this is the existing behavior we can leave it alone for now. It definitely sounds like we need a pass once the migration is complete to shore up security though.